n2v/grid/grider.py

```diff
@@ -225,46 +226,34 @@ def on_grid_ao(self, coeff, grid=None, vpot=None, cubic_grid=False):
 
             if coeff.ndim == 1:
                 l_mat = coeff[(lpos[:])]
-                coeff_r[offset - b_points : offset] = mat_r = contract('pm,m->p', phi, l_mat)
+                coeff_r[offset - b_points : offset] = contract('pm,m->p', phi, l_mat)
```

Comment:
Didn't think python would allow this... So it was redefining mat_r with each block too?